lib/src/client.dart

```diff
@@ -97,7 +97,7 @@ class Client extends MatrixApi {
   DateTime? _accessTokenExpiresAt;
 
   // For CommandsClientExtension
-  final Map<String, FutureOr<String?> Function(CommandArgs)> commands = {};
+  final Map<String, CommandExecutionCallback> commands = {};
```

Comment:
I'm a little bit confused how to actually trigger a command by using a Client without a Room. Can you add a test for it?

Reply:
Ah, probably by using `Client.parseAndRunCommand()`, right? A test for a command without a room could still be useful

Reply:
You are right, I will add a test for a non-room related call.

Reply:
I added a couple of tests for non-room level calls with some different test cases.